pandas/core/reshape/concat.py

```diff
@@ -516,9 +516,12 @@ def _get_new_axes(self) -> List[Index]:
 
     def _get_comb_axis(self, i: int) -> Index:
         data_axis = self.objs[0]._get_block_manager_axis(i)
-        return get_objs_combined_axis(
+        comb_axis = get_objs_combined_axis(
```

Comment:
the copy arg should be pushed down thru to get_objs_combined_axis (it goes deeper, keep pushing it down)

Reply:
done